docs/docsource/Operation-Architecture/Zenko_CLI/Monitoring_NFS-SOFS_Ingestion.rst

```diff
@@ -0,0 +1,67 @@
+.. _Monitoring NFS-SOFS Ingestion:
+
+Monitoring NFS/SOFS Ingestion
+=============================
+
+Zenko 1.1 implements an NFS feature that relies on ingesting the
+namespace of the RING's Scale-Out File System (SOFS). For this
+release, the only available metrics reside in log files. SOFS metadata
+ingestion runs on a cron schedule, and users can make a query against
+Cosmos on the kubectl location to discover how and when scheduled NFS
+activities have been executed.
+
+To find the relevant Cosmos pods, enter:
+
+.. code::
+
+   $ kubectl get pods  | grep "rclone"
+
+   nfs-options-cosmos-rclone-1555329600-7fr27                    0/1     Completed   0          7h
+   nfs-options-cosmos-rclone-1555333140-bb5vf                    1/1     Running     0          1h
+
+Use this information to retrieve the logs with a command formatted as follows:
+
+.. code::
+
+   $ kubectl logs -f {{location}}-cosmos-rclone-{{hash}}
+
+Hence:
+
+.. code::
+
+   $ kubectl logs -f my-location-name-cosmos-rclone-84988dc9d4-9dwjl
+
+yields:
```

Comment:
`yields` -> `Yields`?

Reply:
Nah. It's a continuation of the sentence, not the start of a new one.